ExpressApp/models/api-articles-model.js

```diff
@@ -1,7 +1,14 @@
 const db = require("../../db/connection");
 
-exports.getArticle = (id) => {
+exports.getArticle = (id, next) => {
   return db
     .query(`SELECT * FROM articles WHERE article_id=$1`, [id])
-    .then(({ rows }) => console.log(rows));
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "Not Found" });
```

Comment:
instead of creating a custom error here we can use the PSQL error code which comes back when an article doesn't exist.

Reply:
Oh, the reason I didn't do that is that I get no error. It sends back an empty array which sends back a 200.

Reply:
I've just taken it out and yeah, that's what I'm getting. Just a failing test receiving 200 instead of 404 like it expects